ff/params/readme.md

```diff
@@ -1,3 +1,4 @@
 * smirnoff_1_1_0_am1bcc.py -- "bond charge corrections"
 * smirnoff_1_1_0_ccc.py -- "correctable charge corrections"
-* smirnoff_1_1_0_sc.py -- "simple charge"
\ No newline at end of file
+* smirnoff_1_1_0_sc.py -- "simple charge"
+* smirnoff_1_1_0_test_sampling.py -- "decharged model with LJ adjustments for enhanced sampling tests"
```

Comment:
```suggestion
```

Can delete reference to new file, deleted in https://github.com/proteneer/timemachine/pull/524/commits/5ee7dd87a9c4e6e5672140e60d2d3a2c42aed92b